tests/client/helpers.js

```diff
@@ -18,11 +18,12 @@ export function findByTag(root, tag) {
 }
 
 export function stubAddonManager({ getAddon = Promise.resolve({type: 'addon'}) } = {}) {
-  const fakeAddonManager = {};
-  fakeAddonManager.getAddon = sinon.stub().returns(getAddon);
-  fakeAddonManager.install = sinon.stub().returns(Promise.resolve());
-  fakeAddonManager.uninstall = sinon.stub().returns(Promise.resolve());
-  return fakeAddonManager;
+  return {
+    getAddon: sinon.stub().returns(getAddon),
+    install: sinon.stub().returns(Promise.resolve()),
+    uninstall: sinon.stub().returns(Promise.resolve()),
+    addEventListener: sinon.stub(),
```

Comment:
Ah I confused myself I think here, I think this needs a clearer name to distinguish from navigator.mozAddonManager and our addonManager.